packages/synapse-react-client/src/components/SynapseChat/SynapseChatInteraction.tsx

```diff
@@ -1,45 +1,74 @@
 import { useTheme } from '@mui/material'
 import { ColorPartial } from '@mui/material/styles/createPalette'
-import { AgentChatRequest } from '@sage-bionetworks/synapse-types'
-import { useGetAgentChatWithAsyncStatus } from 'src/synapse-queries/chat/useChat'
+import { SkeletonParagraph } from '../Skeleton'
+import { SmartToyTwoTone } from '@mui/icons-material'
 
 export type SynapseChatInteractionProps = {
-  sessionId: string
   userMessage: string
+  chatResponseText?: string
+  scrollIntoView?: boolean
 }
 
 export const SynapseChatInteraction: React.FunctionComponent<
   SynapseChatInteractionProps
-> = ({ sessionId, userMessage }) => {
+> = ({ userMessage, chatResponseText, scrollIntoView = false }) => {
   const theme = useTheme()
-  const agentChatRequest: AgentChatRequest = useMemo(() => {
-    return {
-      chatText: userMessage,
-      sessionId: sessionId,
+  const ref = useRef<HTMLLIElement | null>(null)
+  useEffect(() => {
+    // on mount, scroll into view if instructed
+    if (scrollIntoView) {
+      if (ref.current) {
+        ref.current.scrollIntoView({ behavior: 'smooth' })
+      }
     }
-  }, [userMessage])
-  const { data: chatResponse, isLoading } =
-    useGetAgentChatWithAsyncStatus(agentChatRequest)
+  }, [ref])
 
-  const chatResponseText = chatResponse?.responseBody?.responseText
   return (
     <>
-      <ListItem sx={{ justifyContent: 'flex-end' }}>
-        <ListItemText
-          primary={userMessage}
+      <ListItem
+        ref={ref}
+        sx={{
+          alignSelf: 'flex-end',
+          backgroundColor: (theme.palette.secondary as ColorPartial)[100],
+          borderRadius: '24px',
+          maxWidth: '70%',
+          display: 'block',
+          mb: '5px',
+          p: '8px 12px',
+          wordWrap: 'break-word',
+          width: 'auto',
+        }}
+      >
+        <ListItemText primary={userMessage} />
+      </ListItem>
+      {chatResponseText && (
+        <ListItem
           sx={{
-            backgroundColor: (theme.palette.secondary as ColorPartial)[100],
-            borderRadius: '10px',
-            padding: '10px',
-            maxWidth: '75%',
+            display: 'grid',
+            gridTemplateColumns: '50px auto',
+            columnGap: '0px',
+            justifyItems: 'center',
+            alignItems: 'start',
+            p: '0px',
           }}
-        />
-      </ListItem>
-      {isLoading && <Skeleton width={200} height={100} />}
-      {!isLoading && chatResponseText && (
-        <ListItem>
+        >
+          <Box
+            sx={{
+              p: '3px',
+              mt: '10px',
+              height: '31px',
+              borderRadius: '50%',
+              borderStyle: 'solid',
+              borderWidth: '1px',
+              borderColor: (theme.palette.grey as ColorPartial)[300],
+            }}
+          >
+            <SmartToyTwoTone
+              sx={{ color: (theme.palette.secondary as ColorPartial)[500] }}
```

Comment:
Since these are in `sx` you can just use strings
```suggestion
              borderColor: 'grey.300',
            }}
          >
            <SmartToyTwoTone
              sx={{ color: 'secondary.main' }}
```